astroid/interpreter/objectmodel.py

```diff
@@ -163,7 +163,10 @@ def attr___name__(self):
 
     @property
     def attr___doc__(self):
-        return node_classes.Const(value=self._instance.doc, parent=self._instance)
+        return node_classes.Const(
+            value=self._instance.doc_node.value if self._instance.doc_node else None,
```

Comment:
```suggestion
            value=getattr(self._instance.doc_node, "value", None),
```
Let's try to replace the ifs with `getattr`. The should simplify that.